pkg/cli/generic_flag.go

```diff
@@ -199,31 +210,31 @@ func (val *genericType[T]) Set(str string) error {
 	case *bool:
 		v, err := strconv.ParseBool(str)
 		if err != nil {
-			return errors.Errorf("error parse: %w", err)
+			return errors.Errorf(`must be one of: "0", "1", "f", "t", "false", "true"`)
 		}
 
 		*dest = v
 
 	case *int:
 		v, err := strconv.ParseInt(str, 0, strconv.IntSize)
 		if err != nil {
-			return errors.Errorf("error parse: %w", err)
+			return errors.Errorf("must be 32-bit integer")
```

Comment:
Why aren't you wrapping the error? It's generally a good practice to.

Reply:
The issue was to clarify the error message the user gets when he specify an invalid value. 
Was:

invalid value "ddddd" for TERRAGRUNT_PARALLELISM: error parse: strconv.ParseInt: parsing "ddddd": invalid syntax


Now: 
```
 invalid value "ddddd" for TERRAGRUNT_PARALLELISM: must be 32-bit integer
```

Reply:
I'll approve it, deferring to your judgment, but I feel like we should be able to wrap the invalid syntax error while updating the error message.

Reply:
You're right! Update the code.

Reply:
You're right! Update the code.